### PR TITLE
CU-25xk151: description with step environment variable keys

### PR DESCRIPTION
**Pull request by kanhaiyas1088**

### CODE REVIEW

File: src/steps/service/StepsService.ts [hunk 1]
```diff
@@ -6,8 +6,12 @@ import Version from '../../model/Version';
 import RequestClientWrapper from '../../RequestClientWrapper';
 import GenerateApiError from '../../utils/GenerateApiError';
 import Step from '../model/Step';
+import StepCreateRequest from '../model/StepCreateRequest';
+import StepUpdateRequest from '../model/StepUpdateRequest';
 
 const EXPECTED_GET_STATUS = 200;
+const EXPECTED_POST_STATUS = 201;
```
Comment (lmkerr) [inline src/steps/service/StepsService.ts:13]: should these just be an Enum at this point?

File: src/steps/service/StepsService.ts [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
 import Resource from '../../model/Resource';
 import Version from '../../model/Version';
 import RequestClientWrapper from '../../RequestClientWrapper';
 import GenerateApiError from '../../utils/GenerateApiError';
 import Step from '../model/Step';
+import StepCreateRequest from '../model/StepCreateRequest';
+import StepUpdateRequest from '../model/StepUpdateRequest';
 
 const EXPECTED_GET_STATUS = 200;
```
Comment (ryankazokas) [inline src/steps/service/StepsService.ts:13]: Remove this if it's no longer needed here. It's scoped to this service so it should be fairly easy to update this.

File: src/steps/model/EnvironmentVariableKey.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 export default interface EnvironmentVariableKey {
   defaultValue?: string;
   name: string;
+  description: string;
```
Comment (ryankazokas) [inline src/steps/model/EnvironmentVariableKey.ts:4]: Description is nullable. Can you add the optional flag via the question mark like in default value.

description?: string;

**Review by ryankazokas — CHANGES_REQUESTED** [review]
Just some minor feedback here @kanhaiyas1088 . Thanks for making these updates. Let me know when you are done making this feedback and I'll take one more look.

**Review by ryankazokas — APPROVED** [review]
Perfect. Thanks @kanhaiyas1088 for making these changes

**Pull request merged**